ui/lib/components/conversation/conversation_item.dart

```diff
@@ -224,14 +222,10 @@ class ConversationItemView {
     _checkboxElement.checked = false;
   }
 
-  void enableCheckbox() {
-    _checkEnabled = true;
-    _checkboxWrapper.classes.toggle("hidden", false);
-  }
-
-  void disableCheckbox() {
-    _checkEnabled = false;
-    _checkboxWrapper.classes.toggle("hidden", true);
+  void enableCheckbox(bool enabled) {
+    _checkEnabled = enabled;
+    _checkboxWrapper.classes.toggle("hidden", !_checkEnabled);
+    _contentWrapper.classes.toggle("full-width", !_checkEnabled);
```

Comment:
should there be a `full-width` class in this PR as well?

Reply:
Ah yes the `full-width` classname already exists, just that we weren't toggling it correctly